cmd/liqoctl/cmd/add.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"github.com/liqotech/liqo/pkg/liqoctl/add"
+)
+
+// installCmd represents the generateInstall command.
+func newAddCommand(ctx context.Context) *cobra.Command {
+	var addCmd = &cobra.Command{
+		Use: add.UseCommand,
+	}
+	addCmd.AddCommand(newAddClusterCommand(ctx))
+	return addCmd
+}
+
+func newAddClusterCommand(ctx context.Context) *cobra.Command {
+	installArgs := &add.ClusterArgs{}
+	var addClusterCmd = &cobra.Command{
+		Use:   add.ClusterResourceName,
+		Short: add.LiqoctlAddShortHelp,
+		Long:  add.LiqoctlAddLongHelp,
+		Run: func(cmd *cobra.Command, args []string) {
+			add.HandleAddCommand(ctx, installArgs)
+		},
+	}
+	addClusterCmd.Flags().StringVar(&installArgs.ClusterAuthURL, add.AuthURLFlagName, "",
+		"the AuthURL of the target Foreign Cluster")
+	addClusterCmd.Flags().StringVar(&installArgs.ClusterToken, add.ClusterTokenFlagName, "",
+		"the ClusterToken of the target Foreign Cluster")
+	addClusterCmd.Flags().StringVar(&installArgs.ClusterName, add.ClusterNameFlagName, "",
+		"the ClusterName assigned to the target Foreign Cluster")
+	addClusterCmd.Flags().StringVar(&installArgs.ClusterID, add.ClusterIDFlagName, "",
+		"the ClusterID assigned of the target Foreign Cluster")
```

Comment:
:+1: The parameter will become optional with a future PR where we will implement the ID retrieve. However, ID should always be passable as a parameter since the AuthUrl may be accessible by the cluster, but not by the host where liqoctl is issued.